blockchain/block/header.go

```diff
@@ -129,7 +129,15 @@ func (h *Header) loadFromBlockHeaderCoreProto(pb *iotextypes.BlockHeaderCore) er
 	copy(h.deltaStateDigest[:], pb.GetDeltaStateDigest())
 	copy(h.receiptRoot[:], pb.GetReceiptRoot())
 	if pb.GetLogsBloom() != nil {
-		h.logsBloom, err = bloom.BloomFilterFromBytesLegacy(pb.GetLogsBloom(), 2048, 3)
+		// TODO: move the number(2048, 3) to top as const or params of the struct
```

Comment:
i think adding a comment here documenting that the bloomfilter in header is legacy(2048, 3) would suffice